index.html

```diff
@@ -0,0 +1,17 @@
+//file <!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css" href="styles/style.css">
+
+    </head>
```

Comment:
Your HTML file needs to have exactly 1 `<head>` section, and exactly 1 `<body>` section:

``` html
<!DOCTYPE html>
<html>
   <head>
      // information about your file goes here - things like the link to your CSS file, your <title> block, etc
   </head>

   <body>
      //  the things that you see on the page go here - things like headers, images and paragraph text
   </body>
</html>
```

In your code you have closed the `<head>` section both in line 7 and in line 9. Since the `<title>` elements belong in the `<head>` section, I suggest removing line 7 to fix this.